core-rust/state-manager/src/transaction/builder.rs

```diff
@@ -106,10 +105,9 @@ pub fn create_new_account_intent_bytes(
     let manifest = ManifestBuilder::new()
         .lock_fee(FAUCET_COMPONENT, 100.into())
         .call_method(FAUCET_COMPONENT, "free", args!())
-        .take_from_worktop(RADIX_TOKEN, |builder, bucket_id| {
-            builder.new_account_with_resource(&AccessRule::AllowAll, bucket_id)
-        })
+        .new_account(&AccessRule::AllowAll)
         .build();
+    // TODO(code review): now somehow deposit the worktop to the created account?
```

Comment:
ooo this may be impossible with the current setup since there is no way to get a reference to the new account just created.

I'd either suggest holding off on the PR until the scrypto side gets updated, or ignoring the tests which currently use this functionality, or update the tests which use to use virtual accounts instead of new accounts.

Reply:
AFAIS there is only 1 test relying on this, so I guess it doesn't hurt to ignore it temporarily 👍